ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/ClientSSESession.kt

```diff
@@ -6,11 +6,23 @@ package io.ktor.client.plugins.sse
 
 import io.ktor.client.call.*
 import io.ktor.sse.*
+import io.ktor.util.reflect.*
 import kotlinx.coroutines.*
 import kotlinx.coroutines.flow.*
 
 /**
  * A Server-sent events session.
```

Comment:
Please consider adding a reference on what SSE is on the wiki page

Reply:
it also would be great to highlight where the receiver type is used in the example

Reply:
Updated KDoc to:
A session for handling Server-Sent Events (SSE) from a server.

Example of usage:
```kotlin
client.sse("http://localhost:8080/sse") { // `this` is `ClientSSESession`
    incoming.collect { event ->
        println("Id: ${event.id}")
        println("Event: ${event.event}")
        println("Data: ${event.data}")
    }
}
```

**To learn more, see [the SSE](https://en.wikipedia.org/wiki/Server-sent_events) and [the SSE specification](https://html.spec.whatwg.org/multipage/server-sent-events.html).**